sources/packages/backend/libs/utilities/src/date-utils.ts

```diff
@@ -307,3 +309,16 @@ export function formatDate(
   }
   return dayjs(date).format(dateFormat);
 }
+
+/**
+ * Checks if the end date is after or same as the start date.
+ * @param startDate start date.
+ * @param endDate end date.
+ * @returns true if end date is after or same as the start date, otherwise, false.
+ */
+export const isSameOrAfterDate = (
```

Comment:
Can't we have any difference in hours or milliseconds for the dates we are comparing?

Reply:
I belive, its date, 
![image](https://github.com/bcgov/SIMS/assets/77353155/7f48c8a0-5da0-4def-bd68-4263631c5762)